src/main/java/ch/wisv/areafiftylan/products/service/TicketService.java

```diff
@@ -66,9 +66,9 @@ default Ticket requestTicketOfType(String type, List<String> options) {
      * Sets up the ticket for transfer
      *
      * @param ticketId     The ID of the ticket to be transferred
-     * @param goalUserName The name of the user which should receive the ticket
+     * @param goalEmail    The name of the user which should receive the ticket
      */
-    TicketTransferToken setupForTransfer(Long ticketId, String goalUserName);
+    TicketTransferToken setupForTransfer(Long ticketId, String goalEmail);
```

Comment:
While we're changing names, doesn't `String receiverEmail` make more sense here?